coalib/results/result_actions/ResultAction.py

```diff
@@ -12,6 +12,9 @@ class ResultAction:
 
     SUCCESS_MESSAGE = 'The action was executed successfully.'
 
+    def __init__(self):
+        self.next_actions = []
+
     @staticmethod
     def is_applicable(result, original_file_dict, file_diff_dict):
```

Comment:
We need to consider for the new design if we still need this or in what degree. I currently imagine that `ResultAction` classes with the new design will perform necessary checks of runtime-values a bear provides at instantiation. Just you are aware ;)